test/test_utils.sh

```diff
@@ -343,11 +399,17 @@ dump_logs() {
     dump_fission_logs $ns $fns router
     dump_fission_logs $ns $fns buildermgr
     dump_fission_logs $ns $fns executor
+    dump_fission_logs $ns $fns storagesvc
     dump_function_pod_logs $ns $fns
     dump_builder_pod_logs $bns
     dump_fission_crds
 }
 
+echo_log() {
```

Comment:
why not just call it `log`? `echo_log` feels kinda odd to write.

Reply:
yeah, now that you mentioned it. sure, I'll change it to log.